### PR TITLE
Etcd clustering

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

**Comment by lebauce** [comment]
run skydive-unit-tests

**Comment by lebauce** [comment]
run skydive-unit-tests

**Comment by lebauce** [comment]
run skydive-unit-tests

**Comment by lebauce** [comment]
run skydive-scale-tests

**Comment by lebauce** [comment]
Closes #364

**Comment by lebauce** [comment]
run unit-tests

**Comment by lebauce** [comment]
run skydive-python-tests

**Comment by lebauce** [comment]
run skydive-devstack-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

File: config/config.go [hunk 1]
```diff
@@ -96,14 +96,14 @@ func init() {
 
 	cfg.SetDefault("etcd.data_dir", "/var/lib/skydive/etcd")
 	cfg.SetDefault("etcd.embedded", true)
-	cfg.SetDefault("etcd.listen", "0.0.0.0:12379")
+	cfg.SetDefault("etcd.name", host)
+	cfg.SetDefault("etcd.listen", "127.0.0.1:12379")
```
Comment (safchain) [inline config/config.go]: localhost:12379 to address ipv6

**Comment by lebauce** [comment]
run devstack-tests

**Comment by eonpatapon** [comment]
Does this means that captures weren't properly shared between analyzers before this ?

(Assuming captures are stored in etcd - not sure about this)

**Comment by lebauce** [comment]
@eonpatapon If you had multiple analyzers and were using the embeded etcd server, only one analyzer had to be set with embedded: true and the others embedded: false

**Pull request merged**